src/types.ts

```diff
@@ -11,24 +11,26 @@ export interface TextAlign {
 // properties
 interface InlineDimensionDef extends EngineAPI.INxInlineDimensionDef {
   textAlign: TextAlign;
+  columnWidth?: ColumnWidth;
```

Comment:
Are these really optional?

Reply:
yes, for the new table, it is optional, remove for now